### DISCUSSION TITLE
Image loading error in Mobile Project Comment

### DISCUSSION THREAD

**Discussion by nfnfgo** [community/community#126625] [Mobile]
As the title says, when using GitHub Project on mobile app (I'm using the iOS one) to view project comment, it seems that the image in the project draft comment could not be loaded properly. Check the image below:

![image](https://github.com/community/community/assets/61616918/2a5e1788-ad69-4baf-98cb-ad6406aef80b)

However with the same device and network I could load the image when switching to GitHub web. 

![image](https://github.com/community/community/assets/61616918/0c082302-5b72-4ff1-aaa8-3b4e721b6a65)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.163.0 (131249690)
Device: iPhone 14 Pro (iOS 17.4.1)
</details>

**Comment by esahmed21** [comment]
Hi @nfnfgo 👋 

Sorry that you are facing this issue, I was able to reproduce for images added to draft issues in a project, I'll create internal issue for our team to investigate it. 
Thank you for reaching out to us!
